### PR TITLE
Add Fair{Run,EventManager}::GetRootManager()

### PR DESCRIPTION
**Pull request by ChristianTackeGSI**
FairRun, and FairEventManager have a FairRootManager linked from them. Expose this, so it can be used.

Start to use it in some few places.

---

Checklist:

* [X] Rebased against `dev` branch
* [X] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [X] Followed [the seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

File: base/steer/FairRun.h [hunk 1]
```diff
@@ -199,6 +199,7 @@ class FairRun : public TNamed
     virtual void SetSource(FairSource* tempSource);
     /** Return non-owning pointer to source **/
     FairSource* GetSource() { return fSource.get(); }
+    FairRootManager* GetRootManager() const { return fRootManager; }
```
Comment (dennisklein) [inline base/steer/FairRun.h:202]:
```suggestion
    FairRootManager& GetRootManager() const { return fRootManager; }
```

I guess, we can make sure to always return one, right?

Comment (ChristianTackeGSI) [reply base/steer/FairRun.h:202]: Yes, we assume that anyway everywhere. And we're quiet sure also.

That said, most of the calling code expects a pointer. That said, let's see, whether we can rewrite them to use references!

Comment (ChristianTackeGSI) [reply base/steer/FairRun.h]: done.

**Review by dennisklein — APPROVED** [review]
Looks good to me.

One thought that comes up still, is "`const` by default". Have you checked if those three variables you touched, could actually be `const`?

**Comment by ChristianTackeGSI** [comment]
done, added const to local variables.

**Pull request merged**